src/engine/GameState.java

```diff
@@ -9,7 +9,7 @@
 public class GameState {
 
 	/** Current game level. */
-	private int level;
+	public static int level;
```

Comment:
Please don't use static variable on `GameState`.
In 2-player-mode, This variable can be different for each instance.